2019-01-03 01:11:47,907 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:48,006 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:48,907 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:49,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:49,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:49,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:50,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:50,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:51,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:51,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:52,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:52,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:53,911 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:53,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:54,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:54,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:55,913 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:55,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:56,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:56,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:57,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:57,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:58,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:58,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:59,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:11:59,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:00,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:00,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:01,914 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:01,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:02,915 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:02,917 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:03,917 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:03,917 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:04,917 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:05,918 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:05,920 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:05,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:06,920 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:06,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:07,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:07,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:08,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:08,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:09,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:09,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:10,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:10,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:11,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:11,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:12,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:12,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:13,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:13,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:14,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:14,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:15,921 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:15,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:16,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:16,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:17,923 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:17,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:18,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:18,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:19,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:19,926 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:20,924 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:21,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:21,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:21,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:22,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:22,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:23,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:23,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:24,927 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:24,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:25,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:25,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:26,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:26,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:27,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:27,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:28,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:28,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:29,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:29,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:30,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:30,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:31,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:31,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:32,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:32,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:33,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:33,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:34,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:34,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:35,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:35,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:36,928 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:37,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:37,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:37,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:38,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:38,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:39,930 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:39,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:40,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:40,933 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:41,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:41,933 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:42,931 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:42,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:43,933 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:43,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:44,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:44,937 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:45,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:45,936 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:46,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:46,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:47,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:47,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:48,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:48,936 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:49,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:49,936 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:50,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:50,936 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:51,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:51,936 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:52,934 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:53,937 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:53,937 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:53,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:54,937 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:54,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:55,937 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:55,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:56,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:56,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:57,938 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:57,940 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:58,940 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:12:58,940 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:14:59,191 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:00,197 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:00,200 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:00,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:01,200 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:01,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:02,201 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:02,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:03,201 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:03,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:04,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:04,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:05,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:05,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:06,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:06,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:07,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:07,206 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:08,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:08,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:09,206 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:09,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:10,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:10,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:11,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:11,206 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:12,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:12,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:13,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:13,206 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:14,203 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:14,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:15,204 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:16,207 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:16,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:16,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:17,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:17,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:18,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:18,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:19,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:19,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:20,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:20,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:21,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:21,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:22,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:22,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:23,211 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:23,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:24,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:24,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:25,210 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:25,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:26,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:26,214 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:27,214 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:27,217 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:28,214 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:28,217 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:29,213 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:29,216 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:30,214 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:30,217 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:31,214 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:32,217 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:32,219 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:32,220 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:33,220 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:33,223 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:34,223 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:34,236 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:35,221 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x0536C9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:15:35,223 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x0536CAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:21:38,694 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_is_outside_lobby_boundary at 0x04C9D9F0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:21:38,697 - chrani-bot:      ERROR @ player thread 76561198040658370: <function player_crossed_location_boundary at 0x04C9DAF0> had an unknown error! (<type 'exceptions.KeyError'>) 
2019-01-03 01:21:58,979 - chrani-bot:      ERROR @ Thread-1: trying to mem on telnet connection failed: [Errno 10053] Eine bestehende Verbindung wurde softwaregesteuert
durch den Hostcomputer abgebrochen / <class 'socket.error'> 
2019-01-03 01:22:56,115 - chrani-bot:      ERROR @ Thread-1: type object 'object' has no attribute 'steamid' <type 'exceptions.AttributeError'> 
2019-01-03 01:37:10,193 - chrani-bot:      ERROR @ Thread-1: type object 'object' has no attribute 'steamid' <type 'exceptions.AttributeError'> 
2019-01-03 13:05:06,338 - chrani-bot:      ERROR @ Thread-1: trying to bc-lp on telnet connection failed: [Errno 10054] Eine vorhandene Verbindung wurde vom Remotehost geschlossen / <class 'socket.error'> 
2019-01-03 14:10:16,976 - chrani-bot:      ERROR @ Thread-1: trying to bc-lp on telnet connection failed: [Errno 10053] Eine bestehende Verbindung wurde softwaregesteuert
durch den Hostcomputer abgebrochen / <class 'socket.error'> 
